zcash_client_backend/CHANGELOG.md

```diff
@@ -6,6 +6,23 @@ and this library adheres to Rust's notion of
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Notable changes
+`zcash_client_backend` now supports TEX (transparent-source-only) addresses as specified
+in ZIP 320. Sending to one or more TEX addresses will automatically create a multi-step
+proposal that uses two transactions. This is intended to be used in conjunction with
+`zcash_client_sqlite` 0.11 or later.
```

Comment:
Technically yes, but the required behaviour from the data API implementation is quite subtle, and I think anyone implementing it would in practice have to look at how `zcash_client_sqlite` does it. How about this:
```suggestion
proposal that uses two transactions. In the case where `zcash_client_sqlite` is used,
version 0.11 or later is needed to take advantage of this functionality.
```

Reply:
That's fine I suppose; it's implicit in the dependency version information of `zcash_client_sqlite` and it's a major version bump to `zcash_client_backend` so I'm not sure that the specific version information is useful _in the changelog_.

Reply:
It doesn't matter, because there are breaking API changes in `zcash_client_backend` such that `zcash_client_sqlite 0.11` will require `zcash_client_backend 0.13`, and the functionality won't be accessible to `zcash_client_sqlite 0.10` (which requires `zcash_client_backend 0.12`) or earlier.

Reply:
```suggestion
proposal that uses two transactions.
```